From 07d4c443e21cd22545d4516515c50b1c8dd39a9b Mon Sep 17 00:00:00 2001
From: Michael Stahl <mstahl@redhat.com>
Date: Tue, 8 Mar 2016 15:01:59 +0100
Subject: sc: remove unnecessary orcus external usage from makefiles

This caused incremental builds to fail after today's orcus upgrade.

The cxx files of these tests don't actually include orcus headers, so
the libraries were not re-linked in incremental builds.

Change-Id: Ibfcf216a7848f3c0adfae4ef59c4b756c290d6b2

diff --git a/sc/CppunitTest_sc_bugfix_test.mk b/sc/CppunitTest_sc_bugfix_test.mk
index 55311cd..712f444 100644
--- a/sc/CppunitTest_sc_bugfix_test.mk
+++ b/sc/CppunitTest_sc_bugfix_test.mk
@@ -16,8 +16,6 @@ $(eval $(call gb_CppunitTest_add_exception_objects,sc_bugfix_test, \
 $(eval $(call gb_CppunitTest_use_externals,sc_bugfix_test, \
 	boost_headers \
 	mdds_headers \
-	orcus \
-	orcus-parser \
 	libxml2 \
 ))
 
diff --git a/sc/CppunitTest_sc_copypaste.mk b/sc/CppunitTest_sc_copypaste.mk
index 7df4f51..641709c 100644
--- a/sc/CppunitTest_sc_copypaste.mk
+++ b/sc/CppunitTest_sc_copypaste.mk
@@ -16,8 +16,6 @@ $(eval $(call gb_CppunitTest_add_exception_objects,sc_copypaste, \
 $(eval $(call gb_CppunitTest_use_externals,sc_copypaste, \
 	boost_headers \
 	mdds_headers \
-	orcus \
-	orcus-parser \
 	libxml2 \
 ))
 
diff --git a/sc/CppunitTest_sc_filters_test.mk b/sc/CppunitTest_sc_filters_test.mk
index 4fbf489..db445f53 100644
--- a/sc/CppunitTest_sc_filters_test.mk
+++ b/sc/CppunitTest_sc_filters_test.mk
@@ -22,8 +22,6 @@ endif
 $(eval $(call gb_CppunitTest_use_externals,sc_filters_test, \
 	boost_headers \
 	mdds_headers \
-	orcus \
-	orcus-parser \
 ))
 
 $(eval $(call gb_CppunitTest_use_libraries,sc_filters_test, \
diff --git a/sc/CppunitTest_sc_opencl_test.mk b/sc/CppunitTest_sc_opencl_test.mk
index 86605a2..ea93111 100644
--- a/sc/CppunitTest_sc_opencl_test.mk
+++ b/sc/CppunitTest_sc_opencl_test.mk
@@ -16,8 +16,6 @@ $(eval $(call gb_CppunitTest_add_exception_objects,sc_opencl_test, \
 $(eval $(call gb_CppunitTest_use_externals,sc_opencl_test, \
 	boost_headers \
 	mdds_headers \
-	orcus \
-	orcus-parser \
 ))
 
 $(eval $(call gb_CppunitTest_use_libraries,sc_opencl_test, \
diff --git a/sc/CppunitTest_sc_rangelst_test.mk b/sc/CppunitTest_sc_rangelst_test.mk
index 2bf6f0f..fa6f690 100644
--- a/sc/CppunitTest_sc_rangelst_test.mk
+++ b/sc/CppunitTest_sc_rangelst_test.mk
@@ -16,18 +16,12 @@ $(eval $(call gb_CppunitTest_use_externals,sc_rangelst_test, \
     icuuc \
 	libxml2 \
 	mdds_headers \
-	orcus \
-	orcus-parser \
 ))
 
 $(eval $(call gb_CppunitTest_add_exception_objects,sc_rangelst_test, \
     sc/qa/unit/rangelst_test \
 ))
 
-$(eval $(call gb_CppunitTest_use_library_objects,sc_rangelst_test, \
-	scqahelper \
-))
-
 $(eval $(call gb_CppunitTest_use_libraries,sc_rangelst_test, \
     basegfx \
     comphelper \
@@ -45,6 +39,7 @@ $(eval $(call gb_CppunitTest_use_libraries,sc_rangelst_test, \
     sax \
     sb \
     sc \
+    scqahelper \
     sfx \
     sot \
     svl \
diff --git a/sc/CppunitTest_sc_subsequent_export_test.mk b/sc/CppunitTest_sc_subsequent_export_test.mk
index 1cfaf69..6c2c9e5 100644
--- a/sc/CppunitTest_sc_subsequent_export_test.mk
+++ b/sc/CppunitTest_sc_subsequent_export_test.mk
@@ -16,8 +16,6 @@ $(eval $(call gb_CppunitTest_add_exception_objects,sc_subsequent_export_test, \
 $(eval $(call gb_CppunitTest_use_externals,sc_subsequent_export_test, \
 	boost_headers \
 	mdds_headers \
-	orcus \
-	orcus-parser \
 	libxml2 \
 ))
 
diff --git a/sc/CppunitTest_sc_subsequent_filters_test.mk b/sc/CppunitTest_sc_subsequent_filters_test.mk
index 08dedbb..845206b 100644
--- a/sc/CppunitTest_sc_subsequent_filters_test.mk
+++ b/sc/CppunitTest_sc_subsequent_filters_test.mk
@@ -16,8 +16,6 @@ $(eval $(call gb_CppunitTest_add_exception_objects,sc_subsequent_filters_test, \
 $(eval $(call gb_CppunitTest_use_externals,sc_subsequent_filters_test, \
 	boost_headers \
 	mdds_headers \
-	orcus \
-	orcus-parser \
 	libxml2 \
 ))
 
-- 
cgit v0.10.2

